web/src/app/modules/shared/components/presentation/modal/modal.component.scss

```diff
@@ -5,3 +5,9 @@
 app-view-container {
   width: 100%;
 }
+
+cds-modal-content {
+  ::ng-deep ngx-monaco-editor {
```

Comment:
I am not really sure what's this doing

Reply:
In #2354 , the editor takes on the height of the parent element. As a result, certain cases (like usage in a modal) leaves the editor unusable. This forces the editor to have a height based on the viewport.